### ISSUE TITLE
help needed with native build

### ISSUE THREAD

**Issue by starius** [mxe/mxe#1046]
I'm adding package `qbittorrent` to plugin `apps`. It depends on file GeoIP.dat (see also #976). I have added package  `geoip-database` with only native target, which installs this file. I don't know how to tell qbittorrent to install native target of `geoip-database`.

My branch: https://github.com/mxe/mxe/compare/master...LuaAndC:torrent2

**Comment by tonytheodore** [comment]
> I don't know how to tell qbittorrent to install native target of geoip-database.

At the moment, you can't. You remove it from the deps and add `$(PKG)_TARGETS  := $(BUILD)` to geoip-database. It will be built along with the other native packages and always be available.

This has the [unfortunate consequence](https://github.com/mxe/mxe/issues/516#issuecomment-161567909) that a change in geoip-database (or lua for that matter) will cause a rebuild of gcc.

**Comment by starius** [comment]
I'm building my [torrent2 branch](https://github.com/mxe/mxe/compare/master...LuaAndC:torrent2) with the following command:
```
$ make MXE_PLUGIN_DIRS=plugins/apps/ qbittorrent
...
make: *** No rule to make target `/home/mxe/mxe-torrent2/usr/i686-w64-mingw32.static/installed/geoip-database', needed by `/home/mxe/mxe-torrent2/usr/i686-w64-mingw32.static/installed/qbittorrent'.  Stop.
```

**Comment by tonytheodore** [comment]
Two options-
 1. remove `geoip-database` from `qbittorrent.mk` deps.
 2. add `$(PKG)_TARGETS  := $(BUILD) $(MXE_TARGETS)` to `geoip-database.mk`

The first assumes native deps are already present, the second cause an empty build rule to be created for cross-targets.

The ideal is something like #516, the options for syntax are:
```
$(PKG)DEPS := geoip-database_$(BUILD)
$(PKG)DEPS := geoip-database~$(BUILD)
$(PKG)DEPS := $(BUILD)~geoip-database
```
Any preference?

**Comment by starius** [comment]
I have chosen the second one.

**Comment by tonytheodore** [comment]
> I have chosen the second one.

Good choice, explicit deps are better.
 
Closed by #1050.
